Dockerfile

```diff
@@ -1,5 +1,3 @@
-ARG ARCH=amd64
```

Comment:
This is intentional.  It seems TARGETARCH is specially handled in the `FROM` during build time